release-notes.txt

```diff
@@ -0,0 +1,82 @@
+BISDN Linux
+4.2.0 Release Notes
+Date: 2021-10-06
+
+! New in this release
+Add EdgeCore AS7726-32X (32 port 100G switch based on Trident3) as supported platform .
+
+! Bug fixes
+For systemd-networkd, the redundant `/etc/systemd/network/99-enp.network` management interface configuration file was dropped, in favor of
+the pre-packaged `/lib/systemd/network/80-wired.network`. For upgrades from earlier version,
+the file in `/etc/systemd/network` wont be removed. Refer to https://docs.bisdn.de/tools/onie-tools.html#onie-bisdn-upgrade.
```

Comment:
```suggestion
the pre-packaged `/lib/systemd/network/80-wired.network`. Note however, If you upgrade from an earlier version
the file in `/etc/systemd/network` will persist. Please see https://docs.bisdn.de/tools/onie-tools.html#onie-bisdn-upgrade
for more information.
```

Reply:
Pherhaps a bit nit-picky, keep it or leave it

Reply:
Well, technically it does not claim anything about any other files, but I think it still might be understood that way.

Reply:
2nd update of suggestion, please review again

Reply:
> Well, technically it does not claim anything about any other files,

"All data is deleted except this specific configuration file" is quite explicit.

Reply:
I already updated the comment